### ISSUE TITLE
Getting the context logits led to a 80% increase in latency

### ISSUE THREAD

**Issue by fc500110** [NVIDIA/TensorRT-LLM#418]
model: llama-7B
GPU: A10

**Comment by byshiue** [comment]
Please share full reproduced steps to help reproduce your issue.

**Comment by byshiue** [comment]
Close this bug because it is inactivated. Please reopen it if needed.
